library/include/borealis/core/font.hpp

```diff
@@ -26,10 +26,14 @@ namespace brls
 
 static constexpr const int FONT_INVALID = -1;
 
-static const std::string FONT_REGULAR        = "regular"; // regular Latin font
-static const std::string FONT_KOREAN_REGULAR = "korean"; // regular Korean font
-static const std::string FONT_MATERIAL_ICONS = "material"; // Material icons font
-static const std::string FONT_SWITCH_ICONS   = "switch"; // Switch icons font (see the HOS shared symbols font for an example)
+static const std::string FONT_REGULAR          = "regular"; // regular font
```

Comment:
Shouldn't we improve the comment here a bit, and explain better what the "regular" font means? Like "The default font to use, will automatically contain fallback for Chinese / Korean extended fonts as needed and icons"